regexes/device/mobiles.yml

```diff
@@ -511,16 +511,21 @@ AllCall:
 
 # Bravis
 Bravis:
-  regex: 'Bravis|A501 Bright|NB(?:10[56]|751|7[145])|NP101'
+  regex: 'Bravis|A501 Bright|NB(?:10[1-8](?:M)?|751|7[145]|8[57]1|76)|NP101|NP 104 3G|B501'
```

Comment:
Wouldn't it actually be the same to write `M?` instead of `(?:M)?`?  Same below

Reply:
A fix was needed for
```
Mozilla/5.0 (Linux; Android 7.0; NB106M Build/NRD90M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/67.0.3396.81 Safari/537.36
```
replace with 106 `(?:106M|10[1-8])   ... )`  ?

Reply:
You can single chars as optional by adding a `?` after it. There is no need to add brackets around (unless it would be more than on char).
If the other devices might have a trailing `M` as well, it wouldn't hurt to to use `(?:10[1-8]M?|...`, otherwise add it as `106M` as you suggested